rest/model/perf_test.go

```diff
@@ -0,0 +1,514 @@
+package model
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	dbmodel "github.com/evergreen-ci/sink/model"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestImportHelperFunctions(t *testing.T) {
```

Comment:
This might be overkill, but I wrote this before writing `TestImport` and realized that I am testing the same thing there. I do like separating these out into unit tests, but up to you as to whether we should keep these or if the tests in`TestImport` is sufficient.